### DISCUSSION TITLE
Blazorise Material: Default Sidebar Menu Links are not rendering on laptop screen but render on mobile size width

### DISCUSSION THREAD

**Discussion by shakeeluetian** [Megabit/Blazorise#2744] [General]
I have configured Blazorise Material Server side application as per instructions of website. However Blazor default Sidebar Menu Links are not visible on my laptop screen but working on mobile width screen. If i comment out ` <link href="css/material.min.css" rel="stylesheet">` Then Sidebar Menu Links are visible but then Material functionality is off-course will not available. Please help me to resolve this issue as i want to use Blazorise Material . I have not done any change in `NavMenu.razor` My `_Host.cshtml` code is as below: 

[_Host.zip](https://github.com/Megabit/Blazorise/files/6896847/_Host.zip)

**Comment by stsrki** [comment]
Default Blazor sidebar template doesn't work well when mixed with Blazorise. Mostly because it overrides some of Bootstrap/Material CSS class names(located in `site.css`). It is recommended to instead use our Bar component as it has much more features and it will look better.

https://blazorise.com/docs/components/bar/

**Reply by shakeeluetian** [reply]
@stsrki i could not succeeded to correctly implement Responsive Expandable and Collapsible Sidebar menu with top header.
actually i am new to programming and not know exactly where to merge code of Bar Component in MainLayout.razor or NavBar.razor. In my different tries there was some kind of issues in correct implementation.

I wish to have  Blazorise Material application having  Blazor Default application layout (responsive sidebar,top header etc) 

Can you provide me a sample "Blazorise Material Application"? Regards

**Reply by stsrki** [reply]
You can see an example in our demo app https://github.com/Megabit/Blazorise/blob/master/Demos/Blazorise.Demo/Layouts/MainLayout.razor

**Reply by shakeeluetian** [reply]
Needful Done. Thanks a lot
